src/amp-tags.js

```diff
@@ -42,10 +42,12 @@ export function StoryAmpTags(seoConfig, config, pageType, data = {}, opts) {
     ? getDomain(currentHostUrl, domainSlug) || config["sketches-host"]
     : "";
   const storySlug = seoConfig.decodeAmpUrl ? decodeURIComponent(story.slug) : encodeURIComponent(story.slug);
+  const { ampPageBasePath = "/amp/story" } = seoConfig;
+  const ampPagePath = typeof ampPageBasePath === "function" ? ampPageBasePath() : ampPageBasePath;
```

Comment:
`ampPageBasePath` need not be a function here.. madrid also will pass it as a string